manifests/docker_registry.pp

```diff
@@ -39,15 +38,15 @@
 
     package {['python-yaml', 'python-ipaddress', 'python-requests']:
         ensure => installed
-    } ->
+    }
 
-    file { "${registry_cleanup_basedir}/clean_registry_cron":
+    -> file { "${registry_cleanup_basedir}/clean_registry_cron":
```

Comment:
We don't use -> to order execution anymore, we have "ordering = manifest" in puppet.conf. That was a "recommended and safe" new setting in Puppet 3.8, but the link to that page is broken now.

I know you didn't introduce them and only reformatted them, but please remove them rather than reformatting them.